### PR TITLE
Audiocodes VoiceAI Connect documentation

### PR DESCRIPTION
**Pull request by znat**
**Proposed changes**:
- Add Audiocodes VoiceAI connect documentation

**Status (please check what you already did)**:
- [ ] added some tests for the functionality
- [ ] updated the documentation
- [ ] updated the changelog (please check [changelog](https://github.com/RasaHQ/rasa/tree/main/changelog) for instructions)
- [ ] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa#code-style) for instructions)

### CODE REVIEW

File: docs/docs/connectors/audioodes-voiceai-connect.mdx [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+<RasaProBanner/>
+
+Use this channel to connect your Rasa assistant to [Audiocodes VoiceAI connect](https://www.audiocodes.com/solutions-products/voiceai/voiceai-connect).
+## Getting Credentials
+
+To get credentials, create a bot on the [VoiceAI connect portal](https://voiceaiconnect.audiocodes.io/).
+
+1. Select **Bots** in the left side bar.
+2. Click on the **+** sign to create a new bot.
+3. Select **Rasa** in the platforms.
```
Comment (chandrikas) [inline docs/docs/connectors/audioodes-voiceai-connect.mdx:23]:
```suggestion
3. Select **Rasa** as the Bot Framework
```

File: docs/docs/connectors/audioodes-voiceai-connect.mdx [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+    token: <token>
+```
+
+You can also specify optional parameters:
+
+| Parameter       | Default value    | Description                                                                                                                 |
+| --------------- | ---------------- | --------------------------------------------------------------------------------------------------------------------------- |
+| `token`         | No default value | The token to authenticate calls between your Rasa assistant and VoiceAI connect                                             |
+| `use_websocket` | `true`           | If `true`, Rasa will send messages through a web socket. If set to `false`, Rasa will send messages through http API calls. |
+| `keep_alive`    | 120              | For each ongoing conversation, VoiceAI will periodically verifies the conversation is still active on the Rasa side.        |
```
Comment (chandrikas) [inline docs/docs/connectors/audioodes-voiceai-connect.mdx:41]:
```suggestion
| `keep_alive`    | 120              | For each ongoing conversation, VoiceAI Connect will periodically verify the conversation is still active on the Rasa side.        |
```

File: docs/docs/connectors/audioodes-voiceai-connect.mdx [hunk 3]
```diff
@@ -0,0 +1,186 @@
...
+  steps:
+      - intent: vaig_event_start
+      - action: utter_greet
+```
+
+Check the [VoiceAI Connect documentation](https://techdocs.audiocodes.com/voice-ai-connect/#VAIG_Combined/voiceai_connect.htm?TocPath=VoiceAI%2520Connect%257C_____0) for an exhaustive list of events.
+
+### Configuring calls
+
+You can send events from Rasa to VoiceAI to make changes to the current call configuration.
```
Comment (chandrikas) [inline docs/docs/connectors/audioodes-voiceai-connect.mdx:90]:
```suggestion
You can send events from Rasa to VoiceAI Connect to make changes to the current call configuration.
```

File: docs/docs/connectors/audioodes-voiceai-connect.mdx [hunk 4]
```diff
@@ -0,0 +1,186 @@
...
+For example, you might want to receive a notification when the user stays silent for more than 5 seconds, or you might need to customize how DTMF digits are sent by VoiceAI Connect.
+
+Call configuration events are sent with custom messages and are specific to the current conversation (sometimes to a message).
+Which means they must be part of your stories or rules so the same behaviour is applied to all conversations.
+
+There is a lot of different events which are exhaustively documented in the [VoiceAI Connect documentation](https://techdocs.audiocodes.com/voice-ai-connect/#VAIG_Combined/voiceai_connect.htm?TocPath=VoiceAI%2520Connect%257C_____0).
```
Comment (chandrikas) [inline docs/docs/connectors/audioodes-voiceai-connect.mdx:96]:
```suggestion
All the supported events are exhaustively documented in the [VoiceAI Connect documentation](https://techdocs.audiocodes.com/voice-ai-connect/#VAIG_Combined/voiceai_connect.htm?TocPath=VoiceAI%2520Connect%257C_____0).
```

File: docs/docs/connectors/audioodes-voiceai-connect.mdx [hunk 5]
```diff
@@ -0,0 +1,186 @@
...
+We will look at one example ehere so you understand how these work and how to use them.
```
Comment (chandrikas) [inline docs/docs/connectors/audioodes-voiceai-connect.mdx:97]:
```suggestion
We will look at one example here to illustrate the use of custom messages and events.
```

File: docs/docs/connectors/audioodes-voiceai-connect.mdx [hunk 6]
```diff
@@ -0,0 +1,186 @@
...
+[`noUserInput` event](https://techdocs.audiocodes.com/voice-ai-connect/#VAIG_Combined/inactivity-detection.htm?TocPath=Bot%2520integration%257CReceiving%2520notifications%257C_____3)
+```yaml
+  utter_config_no_user_input:
+  - custom:
+      type: event
+      name: config
+      sessionParams:
+        userNoInputTimeoutMS: 5000 # If user stays silent for 5 seconds or more, the notification will be sent
+        userNoInputRetries: 2 # If you want to allow for more than one notification during a call
+        userNoInputSendEvent: true # Enable the noUserInput notification
```
Comment (l2nguyen) [inline docs/docs/connectors/audioodes-voiceai-connect.mdx:124]:
```suggestion
        # If user stays silent for 5 seconds or more, the notification will be sent
        userNoInputTimeoutMS: 5000
        # If you want to allow for more than one notification during a call
        userNoInputRetries: 2
        # Enable the noUserInput notification
        userNoInputSendEvent: true
```

**Review by l2nguyen — COMMENTED** [review]
Added some questions based on my confusion when reading this. And then minor formatting changes for readability.

**Review by chandrikas — APPROVED** [review]
Looks good!

**Review by l2nguyen — APPROVED** [review]
LGTM

**Pull request merged**